src/getctydata.c

```diff
@@ -157,14 +157,20 @@ int getpfxindex(char *checkcallptr, char **normalized_call) {
     return w;
 }
 
+/* lookup dxcc country and prefix information from callsign */
+prefix_data *getctyinfo(char *call) {
+    int w = getpfxindex(call, NULL);
+    return prefix_by_index(w);
+}
+
 /* lookup dxcc cty number from callsign */
-int getctynr(char *checkcall) {
+int getctynr(char *call) {
     int w;
 
-    w = getpfxindex(checkcall, NULL);
+    w = getpfxindex(call, NULL);
 
     if (w >= 0)
-	return prefix_by_index(w)->dxcc_index;
+	return prefix_by_index(w)->dxcc_ctynr;
```

Comment:
Renaming `dxcc_index` to `dxcc_ctynr` adds a slight asymmetry to the naming.

On the other hand `countrynr = pfx->dxcc_ctynr` (line 196) is improved. Not sure which is better.

Reply:
I totally agree and see the discrepancy myself. 
While technically it is only an index into the dxcc array we use these index logically as country number. 
I opted for a naming according to the logical meaning. Maybe you have a better idea. Feel free to step in.